tests/nlp/unit/imbu_models_tests.py

```diff
@@ -40,6 +47,94 @@ def setUp(self):
     self.dataPath = os.path.join(TEST_DATA_DIR, "sample_reviews_subset.csv")
 
 
+  def testCreateModel(self):
+
+    # Setup fake ImbuModels instance
+    imbu = ImbuModels(
+      cacheRoot="fake_cache_root",
+      dataPath=self.dataPath,
+      retina="en_associative",
+      apiKey=os.environ.get("CORTICAL_API_KEY")
+    )
+
+    # You must specify a known model type
+    self.assertRaises(Exception, imbu.createModel, "random model type")
```

Comment:
I don't think this testing for the right exception. Should be `self.assertRaises(ValueError, imbu.createModel, modelType, None, None)`